src/components/CartItem/CartItem.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import trashImg from "../../assets/images/trash.png";
+import trashImg_active from "../../assets/images/trash-active.png";
+import Store from "../../contexts/Store";
+import "./cartitem.css";
+import Counter from "../Counter";
+import PropTypes from "prop-types";
+
+const CartItem = ({ cartItem }) => {
+  const [isHover, setIsHover] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(null);
+  const handlerOnMouseEnter = () => {
+    setIsHover((prevIsHover) => true);
```

Comment:
this functional updates are all over the project an are not required, this is the last comment I'll left related to that because it is starting to look like spam
```suggestion
    setIsHover(true);
```